<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>482</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SolaimanLipi</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SAS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>SolaimanLipi</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MainWindow</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSetupdb"/>
    <addaction name="actionInitial_Adjustments"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionAccounts"/>
    <addaction name="actionJournal"/>
    <addaction name="actionInventory"/>
    <addaction name="actionLedger"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Open Database</string>
   </property>
   <property name="font">
    <font>
     <family>SolaimanLipi</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSetupdb">
   <property name="text">
    <string>Setup Database</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+Return</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDaily_Transactions">
   <property name="text">
    <string>Daily Transactions</string>
   </property>
  </action>
  <action name="actionAccounts">
   <property name="text">
    <string>Accounts</string>
   </property>
   <property name="statusTip">
    <string>List of Accounts</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionJournal">
   <property name="text">
    <string>Journal</string>
   </property>
   <property name="statusTip">
    <string>Daily Transactions</string>
   </property>
   <property name="shortcut">
    <string>Alt+J</string>
   </property>
  </action>
  <action name="actionInventory">
   <property name="text">
    <string>Inventory</string>
   </property>
   <property name="statusTip">
    <string>Inventory stock</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionInitial_Adjustments">
   <property name="text">
    <string>Initial Adjustments</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+I</string>
   </property>
  </action>
  <action name="actionLedger">
   <property name="text">
    <string>Ledger</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
